tutorcredentials/plugin.py

```diff
@@ -59,6 +61,38 @@
 )
 
 
+########################################
+# MFEs
+########################################
+
+learner_record_port = 1990
+
+@MFE_APPS.add()
+def _add_learner_record_mfe(apps: dict[str, MFE_ATTRS_TYPE]) -> dict[str, MFE_ATTRS_TYPE]:
+    apps.update(
+        {
+            "learner-record": {
+                "repository": "https://github.com/openedx/frontend-app-learner-record",
+                "refs": get_github_refs_path("openedx/frontend-app-learner-record"),
+                "port": learner_record_port,
+            },
+        }
+    )
+    return apps
+
+tutor_hooks.Filters.ENV_TEMPLATE_VARIABLES.add_items(
```

Comment:
We shouldn't have to add this variable. We can now write `get_mfe("learner-record")['port']`.